docs/apis/kopflos/tutorials/create-first-api.md

```diff
@@ -0,0 +1,194 @@
+# Create Your First API
+
+This tutorial will guide you through the process of creating a minimal Kopflos API from scratch. The API will handle GET requests for RDF instances of `schema:Person` stored in the backing triplestore.
+
+In addition to _Node.js_, you will need _Docker_ to set up a local triplestore.
+
+
+## Setup
+
+First, create a new directory for your project and navigate into it:
+
+```bash 
+mkdir my-api && cd my-api
+```
+
+## Setting Up the Database
+
+Kopflos requires a database (a SPARQL endpoint) to store RDF resources and the API description. We will use Docker Compose to create a local Oxigraph triplestore.
+
+Create a Docker Compose file:
+```bash
+touch docker-compose.yaml
+```
+
+Add the following content to `docker-compose.yaml`:
+
+```yaml
+version: '2'
+services:
+  oxigraph:
+    image: ghcr.io/oxigraph/oxigraph:0.4.2
+    user: root
+    command: serve --location /data --bind 0.0.0.0:7878
+    ports:
+      - 7878:7878
+    volumes:
+      - ./oxigraph:/data
+```
+Start the database:
+```bash
+docker compose up -d
+```
+Once the database is running, open [http://localhost:7878](http://localhost:7878) to access the database UI.
```

Comment:
```suggestion
Once the database is running, you can open [http://localhost:7878](http://localhost:7878) to access the database UI.
```

Changed so that it's clear that this is not a necessary step for the tutorial.